grpc/binlog/v1alpha/binarylog.proto

```diff
@@ -14,6 +14,11 @@
 // limitations under the License.
 // Service exported by server reflection
 
+// IMPORTANT NOTE:
+// This proto was never stabilized, and is now deprecated.
+// The released proto can be found at:
+// https://github.com/grpc/grpc-proto/blob/master/grpc/binlog/v1/binarylog.proto
```

Comment:
Does the `option deprecated = true` work here?

Reply:
Yes but I was hoping the use of this proto was limited enough to avoid putting this everywhere. It doesn't hurt though. Adding it now.